hawc/apps/epiv2/models.py

```diff
@@ -0,0 +1,322 @@
+import reversion
+from django.db import models
+
+from ..study.models import Study
+from . import constants, managers
+
+
+# TODO: set all max_lengths based on the needs of each field
+class AgeProfile(models.Model):
+    objects = managers.AgeProfileManger()
+
+    name = models.CharField(unique=True, max_length=128,)
+
+    def __str__(self):
+        return self.name
+
+
+class Country(models.Model):
+    objects = managers.CountryManager()
+
+    code = models.CharField(blank=True, max_length=2)
+    name = models.CharField(unique=True, max_length=64)
+
+    class Meta:
+        ordering = ("name",)
+        verbose_name_plural = "Countries"
+
+    def __str__(self):
+        return self.name
+
+
+class MeasurementType(models.Model):
+    objects = managers.MeasurementTypeManager()
+
+    description = models.CharField(max_length=128,)
+
+    def __str__(self):
+        return self.description
+
+
+class StudyPopulation(models.Model):
```

Comment:
For each model , we need to add a few methods to get hooked up into our views. Please add these:

```python
   def get_assessment(self):
        return self.study.get_assessment()
   def get_study(self):
        return self.study
```

And then, let's add some unit-tests to test these out to make sure they work. That way we can test the fixture that you added as well.

If you want, we can pair on getting the tests started once you're ready...

Reply:
Also, for StudyPopulation, we'll need a `BREADCRUMB_PARENT = "study"` b/c we'll have a pure django view for that